lib/spack/spack/cmd/tags.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+import six
+
+import llnl.util.tty.colify as colify
+
+import spack.repo
+import spack.tag
+
+description = "list package tags and or their packages"
```

Comment:
Just a suggestion (not a request):
```suggestion
description = "show tags and list their packages"
```